projects/demo/src/modules/info/browsers/index.less

```diff
@@ -0,0 +1,19 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiTable] {
+    inline-size: 100%;
+}
+
+[tuiTh],
+[tuiTd] {
+    border-inline-start: none;
+    border-inline-end: none;
+}
+
+[tuiTd] {
+    font: var(--tui-font-text-m) !important;
+}
+
+td {
+    inline-size: 18.75rem;
```

Comment:
Why is it not under `[tuiTd]` above?

Reply:
It was existing code so I didn't touch that, Let me transform to attribute selector under `[tuiTd]`.

Reply:
Resolved!